test/runtests.jl

```diff
@@ -68,92 +70,107 @@ end
+            "test_basic_VectorQuadraticFunction_Zeros",
+            "nonconvex",
+        ],
+    )
+
+    # conic
+    MOIT.runtests(
+        model, config,
+        include=["conic", "SecondOrderCone", "Semidefinite", "Exponential", "PowerCone", "Cone"],
+        exclude=[
+            "test_basic_VectorAffineFunction_PositiveSemidefiniteConeSquare", # AssertionError: (m.x_sd[ref2id(vi)]).matrix == -1 src/variable.jl:173
+            "test_basic_VectorOfVariables_PositiveSemidefiniteConeSquare",
+            "test_basic_VectorAffineFunction_LogDetConeTriangle",
+            "test_basic_VectorAffineFunction_NormNuclearCone",
+            "test_basic_VectorOfVariables_NormNuclearCone",
+            "test_basic_VectorAffineFunction_NormSpectralCone",
+            "test_basic_VectorOfVariables_NormSpectralCone",
+            "test_basic_VectorAffineFunction_RootDetConeTriangle",
+            "test_basic_VectorOfVariables_RootDetConeTriangle",
+            "test_basic_VectorAffineFunction_PositiveSemidefiniteConeTriangle", # TODO: implement get ConstraintSet for SAF
+            "test_basic_VectorOfVariables_PositiveSemidefiniteConeTriangle",
+            "test_basic_VectorQuadraticFunction_", # not PSD because of equality
+            "test_quadratic_SecondOrderCone_basic",
+            "test_basic_VectorOfVariables_LogDetConeTriangle", # Mosek.MosekError(1307, "Variable '' (1) is a member of cone '' (0).") src/msk_functions.jl:477
+            "test_conic_LogDetConeTriangle_VectorOfVariables",
+        ],
+    )
+
+    # integer
+    MOIT.runtests(model, config,
+        include=["Integer", "ZeroOne"],
+        exclude=[
+            "Cone", "conic",
+            "test_constraint_ZeroOne_bounds", # Cannot put multiple bound sets of the same type on a variable
+            "test_variable_solve_ZeroOne_with_0_upper_bound",
+            "test_variable_solve_ZeroOne_with_upper_bound",
+            "test_basic_ScalarQuadraticFunction_ZeroOne", # non-PSD quadratic
+        ],
+    )
+
+    # other attribute tests
+    MOIT.runtests(model, config,
+        exclude=[
+            "Cone", "conic", "Integer", "ZeroOne", "basic", "linear",
+            "test_model_ListOfConstraintAttributesSet", # list not properly set
+            "BoundAlreadySet", # TODO throw error if bound already set
+            "test_model_ModelFilter_AbstractVariableAttribute",
+            "test_model_VariableName", # Mosek currently throws when setting twice, not when getting names
+            "test_model_Name_VariableName_ConstraintName",
+            "test_model_duplicate_VariableName",
+            "test_model_VariablePrimalStart", # able to set but not to get VariablePrimalStart
+            "test_objective_qp_ObjectiveFunction_zero_ofdiag", # MOI.ListOfModelAttributesSet
+            "test_objective_set_via_modify",
+            "test_quadratic_nonconvex_constraint_integration",
+            "test_solve_ObjectiveBound_MAX_SENSE_LP", # ObjectiveBound invalid
```

Comment:
The quantity of failing tests here isn't good :(

You should try with
```julia
model = MOI.Bridges.full_bridge_optimizer(
    MOI.Utilities.CachingOptimizer(
        MOI.Utilities.UniversalFallback(MOI.Utilities.Model{Float64}()),
        Mosek.Optimizer(),
    ),
    Float64,
)
MOI.Test.runtests(model, MOI.Test.Config())
```

That should add a cache for all of the names etc.

Reply:
added a third group of tests for this